pkg/assembler/graphql/examples/certify_good.gql

```diff
@@ -0,0 +1,130 @@
+fragment allArtifactTree on Artifact {
+  id
+  algorithm
+  digest
+}
+
+fragment allCertifyGoodTree on CertifyGood {
```

Comment:
do we need to add this to path/neighbors example (if exists)?

Reply:
Yeah, I don't know what our strategy for examples should be. I copied these in so that I could cut&paste the whole thing into the playground and be able to do an ingestion mutation.

Reply:
yea i think mainly i'm using the examples for UI stuff, so as long as nothing breaks , doesn't have to be synchronous